git_machete/client.py

```diff
@@ -1899,6 +1907,7 @@ def checkout_github_prs(self,
         if not current_user and my_opened_prs:
             raise MacheteException(
                 "Could not determine current user name, please check your token.")
+        current_function_name: str = f'{inspect.stack()[0].function}()'
```

Comment:
Wow okay... pls leave this change as is, but also open an issue to apply a similar change to all invocations of `debug()` (maybe even better, let `debug` itself extract the name of the calling method from `stack()[1]` or whatnot?)